File: android-exoplayer/src/main/java/com/brentvatne/receiver/BecomingNoisyListener.java
package com.brentvatne.receiver;

public interface BecomingNoisyListener {

    BecomingNoisyListener NO_OP = new BecomingNoisyListener() {
        @Override public void onAudioBecomingNoisy() {
            // NO_OP
        }
    };

    void onAudioBecomingNoisy();

}

File: android/src/main/java/com/brentvatne/receiver/BecomingNoisyListener.kt
package com.brentvatne.receiver

interface BecomingNoisyListener {
    fun onAudioBecomingNoisy()

    companion object {
        val NO_OP: BecomingNoisyListener = object : BecomingNoisyListener {
            override fun onAudioBecomingNoisy() {
                // No operation
            }
        }
    }
}
